modules/images-using-images-s2i-perl-pulling-images.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-using-images-s2i-perl-pulling-images_{context}"]
+= Pulling images for Perl
+
+Images comes in two options:
+
+* RHEL 8
+* CentOS 7
+
+*RHEL 8 Based Images*
+
+The RHEL 8 images are available through the Red Hat Registry.
+
+.Procedure
+
+* To pull the RHEL 8 image, enter the following command for the version of Perl you want:
++
+.Perl `5.16`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/openshift3/perl-516-rhel8
+----
++
+.Perl `5.20`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/perl-520-rhel8
+----
++
+.Perl `5.24`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/perl-524-rhel8
+----
+
+*CentOS 7 Based Image*
+
+A CentOS image for Perl 5.16 is available on link:quay.io[Quay.io].
+
+.Procedure
+
+* To pull the CentOS 7 image, enter the following command:
++
+[source,terminal]
+----
+$ podman pull openshift/perl-516-centos7
```

Comment:
The four images above are deprecated too, we use registry.redhat.io/rhscl/perl-526-rhel7:latest and registry.redhat.io/rhscl/perl-530-rhel7:latest now